### ISSUE TITLE
version '+' sign of transitive dependencies is not recognized

### ISSUE THREAD

**Issue by itanoss** [sbt/sbt#2062]
I have two projects like thw following.
_Project A_ is based on **gradle** and has many dependencies, and **versions** of some of them are defined with **'+' sign**.
_Project B_ is dependent on _Project A_, which is based on **sbt**. sbt version is 0.13.8 and I declared `addMavenResolverPlugin`.

When I build Project B, there is an error of UNRESOLVED DEPENDENCIES.

```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.google.code.javaparser#javaparser;1.+: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]  Note: Unresolved dependencies path:
[warn]      com.google.code.javaparser:javaparser:1.+
(omit)
```

**Comment by jsuereth** [comment]
The maven resolver plugin does not support this feature in 0.13.8.  

I believe 0.13.9 may have the fix you need, you can try out the 0.13.9-M1 and let me know if we got the fix right.

**Comment by eed3si9n** [comment]
I am closing this due to inactivity, but if this is still relevant let us know with more details (reproduction steps etc).
